curriculum/challenges/english/25-front-end-development/quiz-dom-manipulation-and-click-event-with-javascript/66edd07682767adff3a6231e.md

```diff
@@ -17,439 +17,560 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --answer--
 
-Placeholder answer
+`load`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which code is not a way to trigger a `validate` function when you click on a `button` element?
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+<button onclick="validate()">Validate</button>
+```
 
 ---
 
-Placeholder distractor 2
+```html
+<button id="validate_button">Validate</button>
+<script>document.getElementById("validate_button").addEventListener('click', validate);</script>
+```
 
 ---
 
-Placeholder distractor 3
+```html
+<button id="validate_button">Validate</button>
+<script>document.getElementById("validate_button").onclick = validate;</script>
+```
 
 #### --answer--
 
-Placeholder answer
+```html
+<button id="validate_button">Validate</button>
+<script>document.getElementById("validate_button").click = validate;</script>
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What statement is false about `setTimeout` and `setInterval`?
 
 #### --distractors--
 
-Placeholder distractor 1
+The first parameter is the function callback and the second is the delay duration in milliseconds.
 
 ---
 
-Placeholder distractor 2
+`setTimeout` calls a function once after a delay and `setInterval` call a function continously after an interval delay.
 
 ---
 
-Placeholder distractor 3
+They both return a timer's id that can be stopped, respectively with `clearTimeout(id)` and `clearInterval(id)` functions.
 
 #### --answer--
 
-Placeholder answer
+They both are `navigator` properties aliases.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Using the JavaScript Web animation API, how can you animate a `"square"` identified element to scale down to `0` in one second.
 
 #### --distractors--
 
-Placeholder distractor 1
+```js
+document.getElementById("square").animate(
+    [ { transform: "scale(0)" }, { transform: "scale(1)" }, ],
+    { duration: 1 },
+);
+```
 
 ---
 
-Placeholder distractor 2
+```js
+document.getElementById("square").animate(
+    [ { transform: "scale(1)" }, { transform: "scale(0)" }, ],
+    { duration: 1 },
+);
+```
 
 ---
 
-Placeholder distractor 3
+```js
+document.getElementById("square").animate(
+    [ { transform: "scale(0)" }, { transform: "scale(1)" }, ],
+    { duration: 1000 },
+);
+```
 
 #### --answer--
 
-Placeholder answer
+```js
+document.getElementById("square").animate(
+    [ { transform: "scale(1)" }, { transform: "scale(0)" }, ],
+    { duration: 1000 },
+);
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which method does not exists on an `Animation` object instance?
 
 #### --distractors--
 
-Placeholder distractor 1
+`play()`
 
 ---
 
-Placeholder distractor 2
+`pause()`
 
 ---
 
-Placeholder distractor 3
+`cancel()`
 
 #### --answer--
 
-Placeholder answer
+`stop()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+You have an HTML page with a 100x100 pixel canvas identified by the `"art"` ID, and the following code:
```

Comment:
```suggestion
You have an HTML page with a 100x100 pixel canvas identified by the `"art"` id, and the following code:
```

Reply:
"id" was previously changed to "ID" due to naomi suggestion.
https://github.com/freeCodeCamp/freeCodeCamp/pull/56487/commits/eefd6db096130479512c8e8601e610669b8b03e8

Which one should I apply ? There is one other occurence line 434 :
> timer's id

Reply:
@naomi-lgbt maybe should answer this

Reply:
There are two options. If we want to specifically refer to the attribute, it needs to be in code tags `` `id` ``. If we are referring to the abbreviated *meaning*, "identifier", then it should be `ID`.

Reply:
I modified this in the following commit : https://github.com/hedocode/HACKTOBERFEST-2024-DOM-and-Click-Events-Quiz-56206/commit/b9fca843a8790234a8c6662576e57ab9521f2e0c
Tell me if that suits you.